homeassistant/components/aurora_abb_powerone/sensor.py

```diff
@@ -97,7 +148,16 @@ def update(self):
                 _LOGGER.debug("No response from inverter (could be dark)")
             else:
                 raise error
-            self._state = None
         finally:
+            self._available = bool(self._state is not None)
```

Comment:
No need to store it as an instance variable. Just implement the `available` property as `return self._state is not None`

Reply:
Cheers, Done!